Adding info on Harvester port requirements (#280)
rancher/rancher-docs: docs/integrations-in-rancher/harvester.md

Click [here](../pages-for-subheaders/about-provisioning-drivers.md#node-drivers) for more information on node drivers in Rancher.

### Port Requirements

> The port requirements for the Harvester cluster follow the same guidelines as RKE2 [here](https://docs.rke2.io/install/requirements/#networking).

Reviewer: ```suggestion
The port requirements for the Harvester cluster can be found [here](https://docs.harvesterhci.io/v1.1/install/requirements#networking).
```

In addition, other networking considerations are as follows:

- Be sure to enable VLAN trunk ports of the physical switch for VM VLAN networks.

> - Follow the networking best practices as stated [here](https://docs.harvesterhci.io/v1.0/networking/best-practice/overview/).

Reviewer 2: @guangbochen This is removed in v1.1.0, is there a good alternative?

> - Follow the networking best practices as stated [here](https://docs.harvesterhci.io/v1.0/networking/best-practice/overview/).

Reviewer: ```suggestion
- Follow the networking setup guidance [here](https://docs.harvesterhci.io/v1.1/networking/clusternetwork).
```

> For other port requirements for other guest clusters, such as K3s and RKE1, please see [these docs](https://docs.harvesterhci.io/v1.0/install/requirements/#guest-clusters).

Reviewer 2: @jtravee I guess we can target v1.1 since it's the latest supported Harvester version in Rancher v2.6.9.
```suggestion
For other port requirements for other guest clusters, such as K3s and RKE1, please see [these docs](https://docs.harvesterhci.io/v1.1/install/requirements/#guest-clusters).
```
